fix: consider the cli install directory when resolving plugins (#1838)
Redocly/redocly-cli: .changeset/tender-forks-argue.md

---
"@redocly/openapi-core": patch
"@redocly/cli": patch
---

> Fixed a bug where running a preview command when Realm is not in the `node_modules` of the project failed because one of it's dependencies could not be resolved.

Reviewer: ```suggestion
Fixed an issue where running the `preview` command failed because one of its dependencies could not be resolved.
The issue occurred when Realm was not installed in the `node_modules` of the project.
```
